src/layout/CustomLayout.tsx

```diff
@@ -1,33 +1,47 @@
+/* eslint-disable react/jsx-closing-bracket-location */
+/* eslint-disable react/jsx-indent-props */
+/* eslint-disable react/jsx-indent */
+/* eslint-disable @typescript-eslint/no-unused-vars */
 import React from 'react';
 import Layout from 'antd/lib/layout';
 import { PageHeader } from '@ant-design/pro-layout';
 import { Link, useNavigate } from 'react-router-dom';
+import { Button } from 'antd';
 
 const { Header, Content, Footer } = Layout;
 
 const CustomHeader = () => {
   const navigate = useNavigate();
+  const logoSrc = '../../public/logo.png';
```

Comment:
Can't we just import the image at first and then just use that reference?!

Tell me issues with ` const logoSrc = '../../public/logo.png';`